Content.Server/Atmos/Components/GasTankComponent.cs

```diff
@@ -7,21 +7,17 @@
 using Content.Shared.Atmos;
 using Content.Shared.Atmos.Components;
 using Content.Shared.Audio;
-using Content.Shared.Examine;
-using Content.Shared.Interaction;
 using Content.Shared.Sound;
 using Robust.Server.GameObjects;
-using Robust.Server.Player;
 using Robust.Shared.Audio;
 using Robust.Shared.Containers;
 using Robust.Shared.Player;
-using Robust.Shared.Utility;
 
 namespace Content.Server.Atmos.Components
 {
     [RegisterComponent]
 #pragma warning disable 618
-    public sealed class GasTankComponent : Component, IExamine, IGasMixtureHolder
+    public sealed class GasTankComponent : Component, IGasMixtureHolder
 #pragma warning restore 618
```

Comment:
you can also remove the various `#pragma warning disable/restore 618`s around these, assuming that `IExamine` was the only one